packages-exp/functions-compat/src/index.ts

```diff
@@ -18,19 +18,33 @@
 import firebase from '@firebase/app-compat';
 import { name, version } from '../package.json';
 import { registerFunctions } from './register';
-import * as types from '@firebase/functions-types';
+import { FirebaseFunctions as FunctionsCompat } from '@firebase/functions-types';
+import { Functions as FunctionsExp } from '@firebase/functions-exp';
 
+declare module '@firebase/functions-exp' {
```

Comment:
How do I make sure this gets changed to `'@firebase/functions'` when this gets built for release? The transformer doesn't touch this.

Reply:
Modified the transformer.